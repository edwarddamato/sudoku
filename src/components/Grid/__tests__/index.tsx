import * as React from 'react';
import { Grid } from '../index';
import { create } from 'react-test-renderer';

it('renders correctly', () => {
  const grid = create(<Grid blocks={[]} />).toJSON();
  expect(grid).toMatchSnapshot();
});
