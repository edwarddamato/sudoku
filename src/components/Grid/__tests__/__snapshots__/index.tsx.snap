// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="component_grid"
>
  <div
    className="component_game-block"
  >
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
  </div>
  <div
    className="component_game-block"
  >
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
  </div>
  <div
    className="component_game-block"
  >
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
  </div>
  <div
    className="component_game-block"
  >
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
  </div>
  <div
    className="component_game-block"
  >
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
  </div>
  <div
    className="component_game-block"
  >
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
  </div>
  <div
    className="component_game-block"
  >
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
  </div>
  <div
    className="component_game-block"
  >
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
  </div>
  <div
    className="component_game-block"
  >
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
    <div
      className="component_game-cell"
    >
      <input
        className="game-cell_input"
        maxLength={1}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="0"
        type="text"
        value=""
      />
    </div>
  </div>
</section>
`;
